test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerConfigFactoryTests.cs

```diff
@@ -39,6 +39,24 @@ public void DefaultLanguageWorkersDir()
             Assert.Equal(expectedWorkersDir, configFactory.WorkersDirPath);
         }
 
+        [Fact]
+        public void GetDefaultWorkersDirectory_Returns_Expected()
+        {
+            string assemblyLocalPath = Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath);
+            string defaultWorkersDirPath = Path.Combine(assemblyLocalPath, RpcWorkerConstants.DefaultWorkersDirectoryName);
+            Func<string, bool> testDirectoryExists = path =>
+            {
+                return false;
```

Comment:
Should we also have a test that makes testDirectoryExists return `true`?

Reply:
Test covers both. using `Directory.Exists` to test case for 'true'

Reply:
I see now, thank you.